src/app/core/home/home.component.html

```diff
@@ -10,6 +10,7 @@
           <mat-menu #sessions="matMenu" color="primary" [overlapTrigger]="false" class="primary-colored-background">
             <a mat-menu-item routerLink="sessions/search" routerLinkActive="active" color="primary">Search</a>
             <a mat-menu-item routerLink="sessions/create" routerLinkActive="active">Create</a>
+            <a mat-menu-item routerLink="sessions/search-proposition" routerLinkActive="active"> Search proposition</a>
```

Comment:
Was it confirmed by Frank or Jonathan how to get to the new screen? In the story the dominant phrase is: "Find Resources" so the current 'Search Proposition' under 'Sessions' section does not feel right to me